build/Targets/GenerateCompilerExecutableBindingRedirects.targets

```diff
@@ -17,16 +17,41 @@
     <SuggestedBindingRedirects Include="Microsoft.CodeAnalysis.VisualBasic, Version=0.0.0.0, Culture=neutral, PublicKeyToken=$(PublicKeyToken)">
       <MaxVersion>$(AssemblyVersion)</MaxVersion>
     </SuggestedBindingRedirects>
-    <!-- System.Collections.Immutable and System.Reflection.Metadata have a package version that is only three parts instead
-         of the normal four (such as 1.2.0 instead of 1.2.0.0). Binding redirects that do not have exactly four version parts
-         are ignored and as such, we need the special handling below to ensure we end up with four parts to the build version. -->
-    <!-- Please note that this will break if the package version is ever changed from 3 parts. A better, but much more complicated
-         fix would be to check the number of version parts detected and adjust appropriately. -->
     <SuggestedBindingRedirects Include="System.Collections.Immutable, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <MaxVersion>$(SystemCollectionsImmutableVersion.Split('-')[0]).0</MaxVersion>
+      <MaxVersion>1.2.1.0</MaxVersion>
+    </SuggestedBindingRedirects>
+    <SuggestedBindingRedirects Include="System.Console, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+      <MaxVersion>4.0.1.0</MaxVersion>
```

Comment:
Please add a comment explaining why in the code. This otherwise looks like an oversight.